mmdet/structures/bbox/transforms.py

```diff
@@ -88,12 +90,16 @@ def bbox2roi(bbox_list: List[Tensor]) -> Tensor:
         Tensor: shape (n, 5), [batch_ind, x1, y1, x2, y2]
     """
     rois_list = []
+    box_dim = 4
     for img_id, bboxes in enumerate(bbox_list):
+        if isinstance(bboxes, BaseBoxes):
+            box_dim = bboxes.box_dim
+            bboxes = bboxes.tensor
         if bboxes.size(0) > 0:
             img_inds = bboxes.new_full((bboxes.size(0), 1), img_id)
-            rois = torch.cat([img_inds, bboxes[:, :4]], dim=-1)
+            rois = torch.cat([img_inds, bboxes[:, :box_dim]], dim=-1)
```

Comment:
1. Is there any case that the boxes have more dimension?
2. considering line 102, I think the answer for Q1 is no, then we can simplify the code.

Reply:
I suppose `bboxes` must have `ndim == 2` and have no extra dimension. Then, the code can be simplified as below:

```
    for img_id, bboxes in enumerate(bbox_list):
        bboxes = get_box_tensor(bboxes)
        img_inds = bboxes.new_full((bboxes.size(0), 1), img_id)
        rois = torch.cat([img_inds, bboxes], dim=-1)
        rois_list.append(rois)
```